docs/DeveloperGuide.adoc

```diff
@@ -114,6 +126,7 @@ NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X
 [[Design-Model]]
 === Model component
 
+// TODO: Fix diagram layout
 .Structure of the Model Component
 image::ModelClassDiagram.png[]
```

Comment:
Excellent!

Reply:
Perhaps one area of improvement is to make the numbers on the arrows easier to read. Some of them are overlapping and hard to read.

Reply:
Some things to consider:
- For the arrow pointing to the `Model` interface (not from any class), where should it start from?
- Does the arrow pointing to the `Model` interface from `ModelManager` accurately reflect the relationship between the two? 
- Is `Item` an interface? If not, why are classes like `Internship`, `Resume` etc implementing `Item`?